spec/string-indices-spec.js

```diff
@@ -0,0 +1,17 @@
+const chai = require( 'chai' )
+const { expect } = require( 'chai' )
+const {stringIndices} = require( '../src/string-indices' )
+
+describe( 'stringIndices', () => {
+  'use strict'
+
+  it( 'exists', () => {
+    expect( stringIndices ).to.be.a( 'function' )
+  })
+  it('returns an empty string if the supplied index is longer or shorter than the length of the string.', () => {
+    expect( stringIndices('It is such a lovely day.', -3)).to.equal( '' )
+  })
+  it('returns the word at the specified index.', () => {
```

Comment:
0 isn't the default for strings, though, and the user of this function won't ever interact with an array.